src/foam/swift/SwiftJava.js

```diff
@@ -0,0 +1,103 @@
+foam.CLASS({
+  refines: 'foam.core.Argument',
+  properties: [
+  {
```

Comment:
Use the more compact version of properties for these, since we don't plan on adding extra details. ie.
properties: [ 'swiftType, 'swiftDefaultValue' ]
And for all similar uses as well.